index.php

```diff
@@ -26,13 +26,13 @@
 require_once($CFG->libdir . '/adminlib.php');
 require_once($CFG->dirroot . '/local/codechecker/locallib.php');
 
-$path = optional_param('path', '', PARAM_PATH);
+$pathlist = optional_param('path', '', PARAM_RAW);
```

Comment:
While it's not the end of the world... maybe we should apply for PARAM_PATH validation @ some place below? I was expecting `local_codechecker_clean_path()` to, maybe do that, but it's not, [right now](https://github.com/moodlehq/moodle-local_codechecker/blob/24ee399b2439e4ada87cceff046733b0563386de/locallib.php#L294).

Maybe within the `$paths` loop, first thing...

Reply:
Yes good point, I'll add it!